### PR TITLE
Package jsun

### PR DESCRIPTION
**Pull request by jsun28**
Add jar_jdk_packager package and call it in package.py

### CODE REVIEW

**Comment by pvanderknyff** [comment]
Is this replacing the current jar_packager.py? If so, we should probably delete the old one. Also make sure to update the unit tests.

For the rest, it looks good to me, but I'm not very knowledgeable on jar packaging.

**Comment by jsun28** [comment]
well, my opinion is to keep the old one in case of we want to use it in the future, especially when we get into the signing part.

**Comment by rosswbrown** [comment]
Is the assumption 'jar' command is on the PATH?  Any assertion of this we can add temporarily?

File: packaging/package/package.py [hunk 1]
```diff
@@ -13,6 +11,7 @@
 LOG_FILE = 'packaging_log.txt'
 PACKAGED_EXTENSION = ".taco"
 
+
```
Comment (pvanderknyff) [inline packaging/package/package.py]: nit: extra blank line

File: packaging/package/package.py [hunk 2]
```diff
@@ -23,6 +22,7 @@ def create_parser():
 
     return parser
 
+
```
Comment (pvanderknyff) [inline packaging/package/package.py]: nit: extra blank line

File: packaging/package/package.py [hunk 3]
```diff
@@ -87,5 +86,6 @@ def main():
     else:
         argparser.print_help()
 
+
```
Comment (pvanderknyff) [inline packaging/package/package.py]: nit: extra blank line

**Comment by lukewrites** [comment]
> Is the assumption 'jar' command is on the PATH? Any assertion of this we can add temporarily?

@rosswbrown it might be found in/with `os.environ['PATH']`?

**Comment by jsun28** [comment]
add a check_jdk_environ_variable function to check PATH has jdk or not. is that something you want here? @rosswbrown

File: packaging/package/jar_jdk_packager.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    :type jar_filename: str
+
+    :param dest_dir: destination dir to create jar file
+    :type dest_dir: str
+
+    :return: Boolean
+    """
+
+    if not check_jdk_environ_variable():
+        logger.debug("Error: jdk_create_jar: no jdk set up in PATH environment variable, "
```
Comment (lriggs) [inline packaging/package/jar_jdk_packager.py]: logger.error()

Comment (jsun28) [reply packaging/package/jar_jdk_packager.py]: sent a new PR out for this

**Pull request merged**